cartographer_grpc/client_server_test.cc

```diff
@@ -104,6 +104,7 @@ class ClientServerTest : public ::testing::Test {
         num_event_threads = 1,
         num_grpc_threads = 1,
         map_builder = MAP_BUILDER,
+        uplink_server_address = "",
```

Comment:
since you are touching this part of the code, just modify MAP_BUILDER_SERVER provided by map_builder_server.lua rather than replacing it, i.e.
`MAP_BUILDER_SERVER.num_event_threads = 1` because this differs from default.

Reply:
done